packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Timeline/Simple.1.doc.mdx

```diff
@@ -0,0 +1,39 @@
+import Timeline from '@cerner/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/Timeline/BasicTimeline.jsx';
+import PropsTable from '@cerner/terra-graphs/lib/components/Timeline/Timeline?dev-site-props-table';
+import GraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Timeline/timelineConfigObject?dev-site-codeblock';
+import DataObject from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Timeline/defaultDataset.js?dev-site-codeblock';
+require('details-polyfill')
+
+# Simple Line Graph
+
+This is a basic timeline graph.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-graphs`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Timeline from '@cerner/terra-graphs/lib/components/Line/Timeline';
```

Comment:
Sohuld be corrected to 
**import Timeline from '@cerner/terra-graphs/lib/components/Timeline/Timeline';**

Reply:
fixed here: a5aed21a6d74dc81b9b61192b5dddcadc455a452